python/grass/benchmark/plots.py

```diff
@@ -54,18 +54,28 @@ def nprocs_plot(results, filename=None, title=None):
     Each result can come with a different list of nprocs, i.e., benchmarks
     which used different values for nprocs can be combined in one plot.
     """
+    ylabel = ""
     plt = get_pyplot(to_file=bool(filename))
     axes = plt.gca()
```

Comment:
This needs to be there to clear the plots
```suggestion
    _, axes = plt.subplots()
```